vllm/config/compilation.py

```diff
@@ -715,3 +725,69 @@ def custom_op_log_check(self):
                 enable_str = "enabling" if op[0] == '+' else "disabling"
                 logger.warning_once("Op '%s' %s, %s with '%s' has no effect",
                                     op_name, missing_str, enable_str, op)
+
+
+def _parse_operator_name(op_name: str) -> tuple[str, str, str]:
```

Comment:
I don't think this code should live in config, I think the `vllm/compilation` folder is a better place.